ci/docker/debug-cuda.yaml

```diff
@@ -18,15 +18,17 @@ spack:
       true
 
   specs:
-    - dla-future@master build_type=Debug +cuda +miniapps +ci-test ^openblas threads=openmp ^mpich
+    - dla-future@master build_type=Debug +cuda +miniapps +ci-test
```

Comment:
Is
```
    blas:
      require: 'openblas threads=openmp'
    lapack:
      require: 'openblas threads=openmp'
```
missing in this YAML file? I think this now defaults to `intel-mkl`.

Reply:
Ups...

Reply:
This is now fixed.